src/app/modules/gb-data-selection-module/constraint-components/gb-concept-constraint/gb-concept-constraint.component.ts

```diff
@@ -250,9 +254,11 @@ export class GbConceptConstraintComponent extends GbConstraintComponent implemen
   }
 
   set selectedConcept(value: Concept) {
-    (<ConceptConstraint>this.constraint).concept = value;
-    this.initializeConstraints();
-    this.update();
+    if (value instanceof Concept) {
```

Comment:
is it needed when the type of the `value` parameter is Concept?

Reply:
Not having this caused issues, I think with the autocomplete. So, probably the type should be `any` then.

Reply:
```ERROR in src/app/modules/gb-data-selection-module/constraint-components/gb-concept-constraint/gb-concept-constraint.component.ts(252,7): error TS2380: 'get' and 'set' accessor must have the same type.
src/app/modules/gb-data-selection-module/constraint-components/gb-concept-constraint/gb-concept-constraint.component.ts(256,7): error TS2380: 'get' and 'set' accessor must have the same type.```

Reply:
ah ok, I see